cmd/shimdiag/shimdiag.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/Microsoft/go-winio"
+	"github.com/containerd/ttrpc"
+	"github.com/urfave/cli"
+	"golang.org/x/sys/windows"
+)
+
+const (
+	shimPrefix = `\\.\pipe\ProtectedPrefix\Administrators\containerd-shim-`
```

Comment:
Would be good to centralize this logic so if we change the pipe naming it doesn't break this tool. Not needed in this PR though I think.

Reply:
Agreed